bin/pubsub_cli.py

```diff
@@ -120,6 +122,15 @@ def publish(ctx, project_id, topic_name, crashids):
     )
     topic_path = publisher.topic_path(project_id, topic_name)
 
+    # Pull crash ids from stdin if there are any
+    if not crashids and not sys.stdin.isatty():
+        crashids = list(click.get_text_stream("stdin").readlines())
```

Comment:
It took me a while to figure out how to do this and get the right behavior such that it doesn't _block_ on stdin when it shouldn't. Then I switched everything to use click which makes it a lot easier.